### ISSUE TITLE
"This listing does not ship to your selected address"

### ISSUE THREAD

**Issue by rhcastilhos** [OpenBazaar/openbazaar-desktop#618]
V. 2.0.6
Accompanying the message "This listing does not ship to your selected address" there should be a list of where this item can be shipped. Otherwise the buyer has to go back to the listing to see where shipping is available. 

![why shiping](https://user-images.githubusercontent.com/10789865/29007899-85998636-7ae3-11e7-8f94-e3be00dab9e6.png)

**Comment by morebrownies** [comment]
The standard experience in eCommerce sites tends to be what we have in place now, e.g. it says when a listing can't ship to your location and requires you to go back to the listing detail screen to see more information on shipping. 

Here's how eBay handles it: 

![screenshot 2017-08-07 09 56 22](https://user-images.githubusercontent.com/13066886/29037033-bde431a8-7b56-11e7-8833-abe46f9dcb0b.png)

I agree that it's not the best experience, but I don't think outputting the full list of countries the listing ships to within the checkout process is really a great experience either. That would introduce a lot of clutter and if you already have your shipping address set, it's very unlikely that you'll be able to have the item shipped to an alternative country.

**Comment by jjeffryes** [comment]
@morebrownies I think it might be a good idea to add a "Does not ship to any of your current addresses" message in the first section, like we had in v1. That would fix the problem, and prevent frustration from people that try to buy something and don't find out they can't until they're in the purchase process.

**Comment by morebrownies** [comment]
Yeah so change the message from: `This listing does not ship to your selected address` to `This listing does not ship to any of your addresses`?

There would need to be multiple error states then.

**Comment by rhcastilhos** [comment]
The text "This listing does not ship to your selected address" is confusing. It should state that this listing does not ship to user's country.

And I agree with jeffryes, the best to do would be something similar to what Amazon does. 

If buyer address is in a country where shipping isn't available, Amazon shows the following text on the product page:
"_This item does not ship to Brazil. Please check other sellers who may ship internationally. Learn more_".

But user is still allowed to click on Add to Cart button on prodcut page. If user proceed to order page anyway, it shows:
"_Sorry, this item can't be shipped to your selected address. Learn more. You may either change the shipping address or delete the item from your order._ "
